docs/medical-api/more-info/filters.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: "Filtering data"
+description: "How to filter data in Metriport."
+---
+
+Some features include the ability to filter data. In those cases, the syntax below can be used:
+- `unquoted text`: all words are required in to be present in the results, in any order.
+- `"quoted text"`: text inside quote marks will be expected to be included in the results as provided.
+- `or`: allows for optional words/parameter.
+- `-`: excludes the word/parameter following the minus sign.
+
+Examples:
```

Comment:
One thing to clarify - when filtering the only things we can combine are firstname and lastname correct? Like we cant do "Jorge Orta 01/04/1995"? 

Also what kind of dates are valid when filtering?

Reply:
Also would filters be better as an object instead of string? That way we can filter across multiple values (if this isnt apart of the v0 totally fine just wanted to bring it up in case it wasnt considered)

Reply:
Can use any combination of the main fields of the Patient as indicated here: https://github.com/metriport/metriport/pull/3024/files#diff-f79e27fb9e3a766118b8f1abf6e49a26d9e28103cdfc4d24646c4350b617a15eR18

Reply:
Dates are valid using ISO format, prob should mention this and have some examples.

Reply:
> would filters be better as an object instead of string

We can filter across any of the fields that were included in the FTS (full text search) - [ref](https://github.com/metriport/metriport/pull/3024/files#diff-44f2788ec725fb759ad2e203cbb36a7a528ddb742e924ec262804611b5e1ab38R12)

Reply:
Okay great